virtcontainers/interfaces.go

```diff
@@ -34,6 +35,7 @@ type VC interface {
 	StatusContainer(sandboxID, containerID string) (ContainerStatus, error)
 	StopContainer(sandboxID, containerID string) (VCContainer, error)
 	ProcessListContainer(sandboxID, containerID string, options ProcessListOptions) (ProcessList, error)
+	UpdateContainer(sandboxID, containerID string, resources specs.LinuxResources) error
```

Comment:
Why do we need this API? I think `sandbox.UpdateContainer` is enough for us.

Reply:
see cli/update_test.go

Reply:
It's OK for now. I think we have lots of exported functions just used for test, hope we can find a way to remove them and keep the test cases at the same time later.
It's another topic though. This part looks good for now.